runner/runners/invoke.go

```diff
@@ -227,29 +232,38 @@ func (inv *Invoker) run(cmd *runner.Command, id runner.RunID, abortCh chan struc
 
 	stdout, err := inv.output.Create(fmt.Sprintf("%s-stdout", id))
 	if err != nil {
-		return runner.FailedStatus(id, fmt.Errorf("could not create stdout: %v", err),
+		msg := fmt.Sprintf("could not create stdout: %s", err)
+		failedStatus := runner.FailedStatus(id, errors.New(msg),
 			tags.LogTags{JobID: cmd.JobID, TaskID: cmd.TaskID, Tag: cmd.Tag})
+		failedStatus.ActionResult = &bazelapi.ActionResult{GRPCStatus: getInternalErrorStatus(msg)}
```

Comment:
do we need a `if runType == runner.RunTypeBazel`?

Reply:
yes, good catch, looks like a few of these are applicable